HealthcareSystem/PatientWebApp/Controllers/FileController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PatientWebApp.DTOs;
+
+namespace PatientWebApp.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FileController : ControllerBase
```

Comment:
What **FileController** should do exactly?

Reply:
I'm currently refactoring code, so it will be different in the next commit.

Reply:
Okay, feel free to inform me, when you are done. So I can look at the code.